src/constants/analytics.ts

```diff
@@ -1,256 +1,178 @@
-                                        validatorUrl: string;
-                                      }
-                                    : T extends AnalyticsEvent.TradeCancelOrder
-                                      ? {}
-                                      : T extends AnalyticsEvent.TradeCancelOrderConfirmed
-                                        ? {
-                                            /** roundtrip time between user canceling an order and confirmation from indexer (client → validator → indexer → client) */
-                                            roundtripMs: number;
-                                            /** URL/IP of node the order was sent to */
-                                            validatorUrl: string;
-                                          }
-                                        : // Notifcation
-                                          T extends AnalyticsEvent.NotificationAction
-                                          ? {
-                                              type: string;
-                                              id: string;
-                                            }
-                                          : T extends AnalyticsEvent.TransferNotification
-                                            ? {
-                                                type: TransferNotificationTypes | undefined;
-                                                toAmount: number | undefined;
-                                                timeSpent:
-                                                  | Record<string, number>
-                                                  | number
-                                                  | undefined;
-                                                txHash: string;
-                                                status: 'new' | 'success' | 'error';
-                                                triggeredAt: number | undefined;
-                                              }
-                                            : never;
+export const AnalyticsUserProperties = unionize(
+  {
+    // Environment
+    Locale: ofType<SupportedLocales>(),
+    Breakpoint: ofType<
+      'MOBILE' | 'TABLET' | 'DESKTOP_SMALL' | 'DESKTOP_MEDIUM' | 'DESKTOP_LARGE' | 'UNSUPPORTED'
+    >(),
+    Version: ofType<string | null>(),
+
+    // Network
+    Network: ofType<DydxNetwork>(),
+
+    // Wallet
+    WalletType: ofType<WalletType | null>(),
+    WalletConnectionType: ofType<WalletConnectionType | null>(),
+    WalletAddress: ofType<EvmAddress | DydxAddress | null>(),
+
+    // Account
+    DydxAddress: ofType<DydxAddress | null>(),
+    SubaccountNumber: ofType<number | null>(),
+  },
+  { tag: 'type' as const, value: 'payload' as const }
+);
+
+export type AnalyticsUserProperty = UnionOf<typeof AnalyticsUserProperties>;
+
+export const AnalyticsEvents = unionize(
+  {
+    // App
+    AppStart: ofType<{}>(),
```

Comment:
One possible issue is that AppStart.payload is now {} instead of undefined like before.

Reply:
Similar for others. Not sure how this affects the events service.

Reply:
actually I think the behavior was the same before, so we should be good